docs/src/prerequisites.md

```diff
@@ -1,5 +1,10 @@
 ## Pre-requisites
 
+1. Local development tools: git, terraform, a terminal.
```

Comment:
Git and Terraform are both product names so they should be capitalised. I would also recommend adding a link to their respective homepage to each too.